CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## edge-19.1.3
+
+* Controller
+  * Added `linkerd install-sp` command to install service profiles into the
+    linkerd control plane
+  * Improved API for ListPods (thanks @alenkacz!)
```

Comment:
For consistency with the next line, let's escape the endpoint name here with backticks.

```suggestion
  * Improved API for `ListPods` (thanks @alenkacz!)
```